src/main/java/core/basesyntax/figure/Circle.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax.figure;
+
+import core.basesyntax.Figure;
+import core.basesyntax.IShape;
+
+public class Circle extends Figure implements IShape {
+    private double radius;
+
+    public Circle(double radius, String name) {
+        this.radius = radius;
+        setName(name);
+    }
```

Comment:
```suggestion
    public Circle(double radius, String name, String color) {
        super(name, color);
        this.radius = radius;
    }
```
same for other child figures